arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -658,6 +658,14 @@
 					<string>NDVIRaster</string>
 				</array>
 			</dict>
+			<dict>
+				<key>displayName</key>
+				<string>Create feature collection layer (Portal item)</string>
```

Comment:
I'm wondering if we should follow the existing naming convention for this sample family?

![image](https://user-images.githubusercontent.com/2257493/83926675-60fa0e00-a73f-11ea-8ae9-8f619e1b8f73.png)

Also, it would be good to put this one alongside those other two in the Features category. And it looks like we haven't capitalized the first letter of the first word in parenthesis.

Reply:
Ewww that's why I didn't found the similar ones in Layers category... Will fix this!